rfc/span_multithreading.md

```diff
@@ -0,0 +1,33 @@
+# Multithreading Span usage.
+
+**Current State:** Draft
+**Author:** [carlosalberto](https://github.com/carlosalberto)
+
+In the OpenTracing specification, there is no mention of **Span** behavior under multithreaded applications, nor clarification of any thread-safety requirement. This proposal is thus intended to be explicit about this specific scenario.
+
+# Background
+
+Different tracing systems have historically taken different decisions regarding this, with many of them making **Spans** (and their operations) thread-safe. Yet, it has been shown lately that a few frameworks have decided to not make **Spans** thread-safe, and instead provide their own ways to handle this scenario. This could in the near future create confusion among both users and tracing authors, as well as incompatibilities among implementations.
+
+# Specification Changes
+
+The `Span` interface adds one or more paragraph detailing the expected behavior of the objects under multithreading scenarios, specially when a single operation may take place through different threads (maybe using a thread pool). `Span` objects should then either:
+
+* Being thread-safe in all its operations (logs, tags, baggage).
```

Comment:
That will be a performance concern for span's OPs. Many existed APM/tracers expected span don't face multiple threads, that is also based on this.